servers/rendering/renderer_rd/shaders/canvas.glsl

```diff
@@ -313,6 +313,14 @@ vec4 light_compute(
 		vec2 uv,
 		vec4 color, bool is_directional) {
 	vec4 light = vec4(0.0);
+	vec3 light_direction = vec3(0.0);
```

Comment:
Is it correct that you calculate light direction here but then never use it in the actual light calculation? or am I overlooking something here?

Reply:
Happy with the explanation clayjohn provided in chat on how this is used.

Reply:
For future reference, it is only used if the user uses ``LIGHT_DIRECTION`` if the user doesn't use it then this will get optimized out